packages/mui-base/src/Accordion/index.ts

```diff
@@ -0,0 +1,10 @@
+export { AccordionRoot as Root } from './Root/AccordionRoot';
```

Comment:
Do not export hooks and contexts. We may revisit them in the future, but for now we decided to export just the components